src/pages/events/[id]/volunteers.tsx

```diff
@@ -1,44 +1,63 @@
 import { useEffect, useState } from "react";
 import { useRouter } from "next/router";
-import { CiCirclePlus } from "react-icons/ci";
 
 import AssigneeBadge from "@/components/Cards/assignee-badge";
+import VolunteerManageForm from "@/components/Forms/volunteer-manage-form";
 import Header, { HeaderProps } from "@/components/Header";
+import Modal from "@/components/Modal";
 import Sidebar from "@/components/sidebar/index";
 import LoadingSpinner from "@/components/UI/loading-spinner";
 import { useEvent } from "@/hooks/events";
 import { User } from "@/types";
 
 export default function Volunteers() {
-  const [volunteers, _setVolunteers] = useState<User[]>([]);
+  const [volunteers, setVolunteers] = useState<User[]>([]);
   const [headerValues, setHeaderValues] = useState<HeaderProps>();
+  const [VolunteerModal, showVolunteerModal] = useState(false);
   const {
     query: { id: eventId }
   } = useRouter();
 
   const { data: event } = useEvent(eventId as string);
```

Comment:
Unused? Should this be used instead of the fetch call that's setting the header values?